tests/basic/sharedv4_svc_test.go

```diff
@@ -1110,7 +1229,7 @@ var _ = Describe("{Sharedv4SvcFunctional}", func() {
 func getTestSv4Contexts(contexts []*scheduler.Context) []*scheduler.Context {
 	var testSv4Contexts []*scheduler.Context
 	for _, ctx := range contexts {
-		if !strings.HasPrefix(ctx.App.Key, "test-sv4-svc") {
+		if !strings.HasPrefix(ctx.App.Key, "test-sv4-svc") && !strings.HasPrefix(ctx.App.Key, "test-sharedv4") {
```

Comment:
This will cause the other tests to fail since they assume that the apps in testSv4Contexts are using "service"?

Reply:
added a new context for sharedv4 app